hardhat.config.js

```diff
@@ -112,6 +113,20 @@ module.exports = {
         }
       }
     },
+    base: {
+      url: 'https://mainnet.base.org',
+      timeout: 100000,
+      verify: {
+        etherscan: {
+          apiKey: BASESCAN_KEY,
+          apiUrl: 'https://api.basescan.org/'
+        }
+      }
+    },
+    base_sepolia: {
+      url: 'https://sepolia.base.org',
+      timeout: 100000
```

Comment:
What's this?

Reply:
It is configuration for testnet. Timeout is set to 100000, we deduced this from our experience.